README.md

```diff
@@ -154,4 +154,20 @@ Steps you can try if the script isn't working:
 
 ## Development
 
-If you want to play with the `restream` code, you will have to [install Rust](https://www.rust-lang.org/learn/get-started) and [setup the reMarkable toolchain](https://github.com/canselcik/libremarkable#setting-up-the-toolchain) to do cross-platform development.
+If you want to play with the `restream` code, you will have to [install Rust](https://www.rust-lang.org/learn/get-started).
+
+There are two ways of building the required restream binary for streaming the reMarkable framebuffer.
+
+### Using docker and the toltec toolchain
+
+You can use the [toltec toolchain docker images](https://github.com/toltec-dev/toolchain) to build a restream binary compatible with the reMarkable.
+
+```
+docker run --rm -v $(pwd):/project -v /project/.cargo -w "/project" ghcr.io/toltec-dev/rust:latest cargo build --release --target=armv7-unknown-linux-gnueabihf
+```
+
+The generated restream binary can be located under `target/armv7-unknown-linux-gnueabihf/release/restream`
```

Comment:
This holds true for both methods.

Reply:
Moved above to reflect this for both approaches.